src/status_im2/contexts/quo_preview/tabs/tabs.cljs

```diff
@@ -28,27 +23,26 @@
      :label             (str "Tab " (inc index))
      :notification-dot? (and unread-indicators? (zero? (rem index 2)))}))
 
-(defn preview-tabs
+(defn view
   []
   (let [state (reagent/atom {:size        32
                              :scrollable? false})]
     (fn []
       [preview/preview-container
-       {:state      state
-        :descriptor descriptor}
-       [rn/view {:padding-bottom 150}
-        [rn/view
-         {:padding-vertical   60
-          :padding-horizontal 20
-          :flex-direction     :row
-          :justify-content    :center}
-         [quo/tabs
-          (merge @state
-                 {:default-active 1
-                  :data           (generate-tab-items (if (:scrollable? @state) 15 4)
-                                                      (:unread-indicators? @state))
-                  :on-change      #(println "Active tab" %)}
-                 (when (:scrollable? @state)
-                   {:scroll-on-press?    true
-                    :fade-end-percentage 0.4
-                    :fade-end?           true}))]]]])))
+       {:state                     state
+        :descriptor                descriptor
+        :component-container-style {:padding-vertical   60
+                                    :padding-horizontal 20}}
+       [rn/view
```

Comment:
I think here you can eliminate this `view` and move its styles to the `component-container-style` you already have above.

Reply:
done